EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/EventGenerator.java

```diff
@@ -1,91 +1,79 @@
 /*******************************************************************************
  *
- * Copyright (c) 2012-2015, Business Process Technology (BPT),
+ * Copyright (c) 2012-2017, Business Process Technology (BPT),
  * http://bpt.hpi.uni-potsdam.de.
  * All Rights Reserved.
  *
  *******************************************************************************/
 package de.hpi.unicorn.application.pages.input.generator;
 
+import de.hpi.unicorn.application.pages.input.generator.attributeInput.AttributeInput;
+import de.hpi.unicorn.application.pages.input.generator.attributeInput.DateAttributeInput;
 import de.hpi.unicorn.application.pages.input.replayer.EventReplayer;
 import de.hpi.unicorn.attributeDependency.AttributeDependency;
 import de.hpi.unicorn.attributeDependency.AttributeDependencyManager;
 import de.hpi.unicorn.attributeDependency.AttributeValueDependency;
 import de.hpi.unicorn.event.EapEventType;
 import de.hpi.unicorn.event.EapEvent;
-import de.hpi.unicorn.event.attribute.AttributeTypeEnum;
 import de.hpi.unicorn.event.attribute.TypeTreeNode;
 import org.apache.log4j.Logger;
 
 import java.io.Serializable;
-import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.text.DateFormat;
 import java.util.ArrayList;
 import java.util.Date;
-import java.util.EnumMap;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.Random;
-import java.util.concurrent.ThreadLocalRandom;
 
 /**
  * Can generate events and push them to replayer.
  */
-public class EventGenerator {
+class EventGenerator {
 
     private static final int DEFAULT_REPLAY_SCALEFACTOR = 1000;
     private int replayScaleFactor = DEFAULT_REPLAY_SCALEFACTOR;
-    private static Random random = new Random();
     private static final DateFormat dateFormatter = new SimpleDateFormat("yyyy/MM/dd'T'HH:mm");
     private static final Logger logger = Logger.getLogger(EventGenerator.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "logger" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1068)